src/Streamly/Internal/FileSystem/FD.hs

```diff
@@ -217,7 +217,11 @@ readArrayUpto size (Handle fd) = do
     -- ptr <- mallocPlainForeignPtrAlignedBytes size (alignment (undefined :: Word8))
     withForeignPtr ptr $ \p -> do
         -- n <- hGetBufSome h p size
+#if __GLASGOW_HASKELL__ < 900
```

Comment:
constraint on base ?

Reply:
This is imported from GHC `import qualified GHC.IO.Device as RawIO`

Reply:
it is part of base https://hackage.haskell.org/package/base-4.14.1.0/docs/GHC-IO-Device.html#t:RawIO

Reply:
ok, will change.